code/obj/item.dm

```diff
@@ -401,7 +401,7 @@
 		SPAWN(0.6 SECOND)
 			if (!src || !M || !user)
 				return
-			SEND_SIGNAL(M, COMSIG_ITEM_CONSUMED, user, src)
+			SEND_SIGNAL(src, COMSIG_ITEM_CONSUMED, user, src)
```

Comment:
You are right, there's only one other thing that uses this signal and I misread it. However, I think the proper fix would be to create a new signal of `COMSIG_MOB_ITEM_CONSUMED` and have the other thing trigger on that, since the rest of the `COMSIG_ITEM` signals trigger on the item, and `COMSIG_MOB` triggers on the mob.

Reply:
In fact, all of the `COMSIG_ITEM_CONSUMED` signals seem wrong, they all target mobs instead of items despite being named like other item signals

Reply:
Fixing this in another PR

Reply:
#9763